dynomite-derive/src/lib.rs

```diff
@@ -115,6 +115,7 @@ fn parse_attrs(all_attrs: &[Attribute]) -> Vec<Attr> {
 /// # Panics
 ///
 /// This proc macro will panic when applied to other types
+//#[proc_macro_error::proc_macro_error]
```

Comment:
note to self: revisit this before merging. it seems the proc_macro_error crate requires this and I didn't catch that last time.